server/plugin/subscriptions.go

```diff
@@ -13,15 +13,15 @@ import (
 )
 
 const (
-	SubscriptionsKey              = "subscriptions"
-	excludeOrgMemberFlag          = "exclude-org-member"
-	excludeOrgReposFlag           = "exclude"
-	SubscribedRepoNotificationOff = "subscribed-turned-off-notifications"
+	SubscriptionsKey     = "subscriptions"
+	excludeOrgMemberFlag = "exclude-org-member"
+	excludeOrgReposFlag  = "exclude"
 )
 
 type SubscriptionFlags struct {
 	ExcludeOrgMembers bool
 	ExcludeOrgRepos   bool
+	ExcludeRepos      []string
```

Comment:
Should this be 
```suggestion
	ExcludedRepos      []string
```

as it's not a bool flag?

Reply:
correct, fixed.